docs/scos/dev/front-end-development/202212.0/oryx/oryx-presets.md

```diff
@@ -37,11 +37,15 @@ export const b2cFeatures: AppFeature[] = [
   ...
 ```
 
+Feature sets also contain static experience data. Experience data includes the structure and layout of the components of the application, such as pages and sections. By utilizing the static experience data provided by the presets, you do no need to set up any boilerplate code. Moreover, we avoid hardcoded page structures that would not allow for personalized experiences going forward.
+
 For more information about feature sets, see [Feature sets](/docs/scos/dev/front-end-development/{{page.version}}/oryx/oryx-feature-sets.html)
 
 ## Themes
 
-The overarching UI of the application is driven by themes. To get started with a default theme, you can import it from the preset package and apply it to your Oryx application:
+A theme represents the global visual appearance of an application, including typography, colors, and other specific design elements, such as form field placeholder color. The theme is build with design tokens, which are configurable values that are used in CSS properties. By configuring these design tokens, you can customize the theme and align the application and it's components with your brand identity or specific design requirements.
```

Comment:
Is this a list? `form field placeholder color`

Reply:
i've rephrased it